docs/terms-policies/governance.md

```diff
@@ -0,0 +1,267 @@
+- Manage user accounts and permissions.
+- Respond to data access requests and issues.
+
+Administrators:
+- @satra, @yarikoptic, @waxlamp, @mvandenburgh, @jjnesbitt, @danlamanna, @brianhelba, @kabilar, @asmacdo, @bendichter, @candleindark, @CodyCBakerPhD.
+
+## 6. Decision-Making Model
+
+### 6.1 Roadmap
+- Project targets are discussed during the biweekly Engineering Core and Scientific Core meetings.
+- Project Leadership provides guidance on prioritization of targets.
+- Public notes of these meetings are available on [Google Drive](https://drive.google.com/drive/folders/1-jXLpcrh3L650FiZyTFgcs096nZjO2C3).
+
+### 6.2 Consensus Process
+1. Open a GitHub issue describing the bug/feature request, context, and possible solutions.
+2. For major architectural changes, create a design document and submit as a PR for discussion and refinement.
+   For reference, see [previous design documents](https://github.com/dandi/dandi-archive/tree/master/doc).
+3. Collect feedback during a 7 day review period.
+4. Summarize consensus in the GitHub issue and resolve all suggestions in the design document.
+5. Implement via pull request(s) referencing the proposed design.
+
+### 6.3 Conflict of Interest
+- Participants should disclose direct commercial interest in a technology choice.
+- Conflicted member recuses oneself from final decision phase.
+
+## 7. Pull Request Workflow
+
+### 7.1 Pull Request Requirements
+- Link the associated issue.
+- Add a clear description (problem, approach, alternatives considered).
+- Major architectural changes require a design document.
+- Add or update tests.
+- Update documentation.
+- Ensure CI passes.
+- Large pull requests should be split unless justified.
+- No introduction of unreviewed secrets or credentials.
+- Verified provenance for large binary additions (discouraged in code repos).
+
+### 7.2 Merge Policy
+- All pull requests require:
+    - All comments must be resolved or addressed.
+    - If a comment cannot be resolved, the Project Leadership would be enlisted to decide on the path forward.
+    - Approval by at least 1 listed Maintainer for that repository.
+    - 24 hour waiting period (unless addressing a critical issue or trivial issue [e.g. typos]).
+- See section below regarding updates to the Governance document.
+
+### 7.3 Draft vs Ready for Review
+- Open as a Draft for early feedback.
+- Convert to “Ready” only when tests and documentation are updated.
+
+### 7.4 Reverts
+- Any Maintainer may revert a merged pull request causing regression, security issue, or service degradation, with immediate notice in original pull request thread.
+- All changes (including reverts) must be submitted through a pull request, and a new release must be made if the prior change was already released.
+- Follow-up issue required to track remediation.
+
+## 8. Releases
+
+### 8.1 Versioning
+- [Semantic Versioning 2.0](https://semver.org/spec/v2.0.0.html) for APIs and libraries.
+
+### 8.2 Release Steps
```

Comment:
this seems more implementation than policy. perhaps that's a consideration in going through this document to separate policy from example implementation. the implementation could be an example of some policy on how we will do it in dandi.